hermeto/core/config.py

```diff
@@ -73,6 +77,7 @@ def settings_customise_sources(
         """
         return (
             init_settings,
+            env_settings,
```

Comment:
Why does `init_settings` have higher priority? What use case did you have in mind where we'd not want to respect user settings?